### DISCUSSION TITLE
What does 'pac' in 'pac cli' stand for?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/powerplatform-build-tools#755] [General]
I know this shouldn't matter, but I'm curious... Everything I can find online (like https://learn.microsoft.com/en-us/power-platform/developer/cli/introduction) refers to "Microsoft Power Platform CLI", but how does 'pac' stand in for 'Microsoft Power Platform'? Is it an acronym for something? Or should I just give up and accept that 'pac' = "Microsoft Power Platform"?

**Comment by petrochuk** [comment]
Even we are not sure :) I think it started as 

**P**ower **A**pps **C**ommand line = pac

but over time expanded to other areas of Power Platform and for obvious compatibility reasons was never renames to MPPCLI

**Comment by rajyraman** [comment]
> I know this shouldn't matter, but I'm curios... Everything I can find online (like [https://learn.microsoft.com/en-us/power-platform/developer/cli/introduction]) refers to "Microsoft Power Platform CLI", but how does 'pac' stand in for 'Microsoft Power Platform'? Is it an acronym for something? Or should I just give up and accept that 'pac' = "Microsoft Power Platform"?

It started out as a CLI for building PCF components. https://powerapps.microsoft.com/en-us/blog/the-powerapps-component-framework-and-the-powerapps-cli-is-now-available-for-public-preview/

Then it transformed into one CLI for everything Power Platform, but exe name stayed the same.

**Reply by ghost** [reply]
Aha! That link about it starting out with PCFs was helpful... based on that I feel confident declaring that "**pac cli**" = "**P**ower **A**pps **C**omponent **C**ommand **L**ine **I**nterface"
